docs/contributing/code_contribution_guide.rst

```diff
@@ -143,25 +156,15 @@ Once you have synced your branch push your changes and check if file conflicts a
 General Tips
 -------------
 
-1. It's good practice to link PR to an issue during commit message. Such as
-stating ``Fix #132`` for fixing issue 132.
-
-2. If you have an issue, ask your question in slack before reporting issue. If
-your issue is not resolved check any open issues for resolution before creating
-a new issue.
+- If you have an issue, ask your question in slack before reporting issue. If your issue is not resolved check any open issues for resolution before creating a new issue.
 
-3. For new features or significant code refactor please notify maintainers and
-open an issue before working on task to keep everyone informed.
+- For new features or significant code refactor please notify maintainers and open an issue before working on task to keep everyone informed.
 
-4. If you open an issue, please respond back during discussion, if there is no
-activity the issue will be closed.
+- If you open an issue, please respond back during discussion, if there is no activity the issue will be closed.
 
-5. Please refrain from opening duplicate issue, check if there is an existing
-issue addressing similar problem, instead you can participate in discussion in
-the issue or contact appropriate individuals directly in slack.
+- Please refrain from opening duplicate issue, check if there is an existing issue addressing similar problem. You can ask questions in slack to report your issue or contact project maintainers.
```

Comment:
Please add "a" after opening. Pluralize problem.

Reply:
fixed

Reply:
Please add "a" after opening and before it.

Reply:
fixed